sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/projects/models/_models_py3.py

```diff
@@ -0,0 +1,1769 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+import datetime
+from typing import Dict, List, Optional, Union
+
+from azure.core.exceptions import HttpResponseError
+import msrest.serialization
+
+from ._question_answering_projects_client_enums import *
+
+
+class ActiveLearningFeedback(msrest.serialization.Model):
```

Comment:
I'm inclined to think we actually try generating this without models. I think that will be cleaner given that we're going with an LLC approach to this side of the SDK.